guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p>"In <code>arcgis.learn</code>, all the discriminators and generators have been grouped into a single model" Please remove this, it is the inherent architecture of cyclegan. Its grouped everywhere.</p><p> </p><p>Describe a little about the objective function or loss that we get combining the three losses discussed above. feels incomplete.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>